### PR TITLE
[AR] week3 -2

### PR DESCRIPTION
**Pull request by ElJAZRY**
Translation for ch03-2 to arabic.

### CODE REVIEW

File: docs/ar/week03/03-2.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+lecturer: Yann LeCun
+title: .الشبكات الالتفافية تطورها, معماريتها, تفاصيل التطبيق البرمجي ومميزاتها
+authors: Chris Ick, Soham Tamba, Ziyu Lei, Hengyu Tang
+date: 10 Feb 2020
+translation data: 20 Aug 2020
+translator: Mohamed Adel Musallam
+
+---
+
+<div dir="rtl">
```
Comment (Atcold) [inline docs/ar/week03/03-2.md:13]: Please, remove these. We are going to handle them at page template level.

File: docs/ar/week03/03-2.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
+---
+lang-ref: ch.03-2
+lang: ar
+lecturer: Yann LeCun
+title: .الشبكات الالتفافية تطورها, معماريتها, تفاصيل التطبيق البرمجي ومميزاتها
+authors: Chris Ick, Soham Tamba, Ziyu Lei, Hengyu Tang
+date: 10 Feb 2020
+translation data: 20 Aug 2020
+translator: Mohamed Adel Musallam
+
```
Comment (Atcold) [inline docs/ar/week03/03-2.md:10]: Remove empty line.

File: docs/ar/week03/03-2.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+---
+
+<div dir="rtl">
+
+# [النماذج الأولية للشبكات الالتفافية وتطورها](https://www.youtube.com/watch?v=FW5gFiJb-ig&t=2949s)
+
+## تطبيق الشبكات العصبية البدائية على مجموعات  من البيانات الصغيرة 
+
+</div>
```
Comment (Atcold) [inline docs/ar/week03/03-2.md:19]: Remove this as well.

**Review by ElJAZRY — COMMENTED** [review]
Update requested changes.

**Review by Atcold — APPROVED** [review]
Thanks!

**Pull request merged**